cli/src/main/resources/hudson/cli/client/Messages.properties

```diff
@@ -1,23 +1,23 @@
-   -remoting    : use deprecated Remoting channel protocol (if enabled on server; for compatibility with legacy commands or command modes only)\n\
-   -i KEY       : SSH private key file used for authentication (for use with -ssh or -remoting)\n\
-   -p HOST:PORT : HTTP proxy host and port for HTTPS proxy tunneling. See https://jenkins.io/redirect/cli-https-proxy-tunnel\n\
-   -noCertificateCheck : bypass HTTPS certificate check entirely. Use with caution\n\
-   -noKeyAuth   : don't try to load the SSH authentication private key. Conflicts with -i\n\
-   -user        : specify user (for use with -ssh)\n\
-   -strictHostKey : request strict host key checking (for use with -ssh)\n\
-   -logger FINE : enable detailed logging from the client\n\
-   -auth [ USER:SECRET | @FILE ] : specify username and either password or API token (or load from them both from a file);\n\
-                                   for use with -http, or -remoting but only when the JNLP agent port is disabled.\n\
-                                   Passing crendentials by a file is recommended.\n\
-                                   See https://jenkins.io/redirect/cli-http-connection-mode for more info and options.\n\
+  \ -s URL              : the server URL (defaults to the JENKINS_URL env var)\n\
+  \ -http               : use a plain CLI protocol over HTTP(S) (the default; mutually exclusive with -ssh and -remoting)\n\
+  \ -ssh                : use SSH protocol (requires -user; SSH port must be open on server, and user must have registered a public key)\n\
+  \ -remoting           : use deprecated Remoting channel protocol (if enabled on server; for compatibility with legacy commands or command modes only)\n\
+  \ -i KEY              : SSH private key file used for authentication (for use with -ssh or -remoting)\n\
+  \ -p HOST:PORT        : HTTP proxy host and port for HTTPS proxy tunneling. See https://jenkins.io/redirect/cli-https-proxy-tunnel\n\
+  \ -noCertificateCheck : bypass HTTPS certificate check entirely. Use with caution\n\
+  \ -noKeyAuth          : don't try to load the SSH authentication private key. Conflicts with -i\n\
+  \ -user               : specify user (for use with -ssh)\n\
+  \ -strictHostKey      : request strict host key checking (for use with -ssh)\n\
+  \ -logger FINE        : enable detailed logging from the client\n\
+  \ -auth [ USER:SECRET | @FILE ] : specify username and either password or API token (or load from them both from a file);\n\
+  \                                 for use with -http, or -remoting but only when the JNLP agent port is disabled.\n\
+  \                                 Passing crendentials by a file is recommended.\n\
+  \                                 See https://jenkins.io/redirect/cli-http-connection-mode for more info and options.\n\
   \n\
-  The available commands depend on the server. Run the 'help' command to\n\
+  The available commands depend on the server. Run the 'help' command to \
```

Comment:
Hmm. why?

Reply:
It's an arbitrary word wrapping location. Other lines above are wider.